### PR TITLE
Update our website for Weblate

### LINKED ISSUE #1613: Update for Weblate
This is going to include copy changes for https://elementary.io/docs/translation-guide#translating-our-website

and also https://elementary.io/get-involved#translations

### LINKED ISSUE #1623: Transifex link gives 404
The instructions on how to translate the website seems not to work anymore:
https://elementary.io/da/get-involved#translations

The links to Transifex gives 404.

### PR DESCRIPTION
**Pull request by danrabbit**
Fixes #1613 
Fixes #1623 

This pull request is ready for review.

### CODE REVIEW

File: get-involved.php [hunk 1]
```diff
@@ -99,11 +99,11 @@
         <div class="half">
             <p>elementary OS is created and used by people from all around the World; help us make the experience even better by translating it into more languages.</p>
             <p>elementary OS uses an open, collaborative translation tool on Launchpad called Rosetta. <a href="/docs/translation-guide#translating-applications" class="read-more">Learn More</a></p>
```
Comment (cassidyjames) [inline get-involved.php]: Are we still using Rosetta just for apps on Launchpad now? Should we be more clear about that?

**Comment by png2378** [comment]
I think we shouldn't publish this until we are convinced that Weblate works well and has no critical issues.

File: get-involved.php (hunk 1 above)
Comment (danrabbit) [inline get-involved.php]: Yeah we never used Transifex for apps either. Transifex only was used for the website

**Comment by btkostner** [comment]
We also have some JS code for the translation button I'm the docs.

**Comment by cassidyjames** [comment]
Yeah, because this replaces Transifex which is a 404 right now, this is critical. Merging.

**Pull request merged**